workflows/versions.json

```diff
@@ -0,0 +1,2 @@
+{
```

Comment:
I think we need to commit a baseline file, so next time an update triggers a valid set of tests.